compose/services/docker-compose.fuelrod-new.yml

```
@@ -33,16 +35,45 @@
     networks:
       - web
       - internal
+    healthcheck:
+      test: [ "CMD", "curl","-f", "http://localhost/health" ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)